packages/lit-helpers/test/read-only-properties-mixin.test.js

```diff
@@ -0,0 +1,39 @@
+import { expect, html, fixture, defineCE, unsafeStatic } from '@open-wc/testing';
+import { LitElement } from 'lit-element';
+import { ReadOnlyPropertiesMixin } from '../src/read-only-properties-mixin.js';
+
+describe('ReadOnlyPropertiesMixin', () => {
+  let element;
+  beforeEach(async () => {
+    const ReadOnlyElement = unsafeStatic(
+      defineCE(
+        class extends ReadOnlyPropertiesMixin(LitElement) {
+          static get properties() {
+            return {
+              publicString: { type: String },
+              readOnlyString: { type: String, readOnly: true },
+            };
+          }
+
+          constructor() {
+            super();
+            this.readOnlyString = 'readOnlyString';
+          }
+        },
+      ),
+    );
+    element = await fixture(html`<${ReadOnlyElement}></${ReadOnlyElement}>`);
+  });
+
+  it('defines a read only property', async () => {
+    const initial = element.readOnlyString;
+    element.readOnlyString = Date.now().toString(36);
+    expect(element.readOnlyString).to.equal(initial);
+  });
+
```

Comment:
what happens if you do `element.readOnlyString = 'foo'`? will it throw? it probably should?

in any way can we test what it does?

Reply:
Silent noop.